lib/lightning_web/live/workflow_live/edit.ex

```diff
@@ -412,6 +401,7 @@ defmodule LightningWeb.WorkflowLive.Edit do
           centered
         />
         <.form
+          :if={@selection_mode !== "expand"}
```

Comment:
@doc-han what was the reasoning for this? We've got some failing tests (_happy to work around this change_), but was curious why we're not rendering the form conditionally here.

Reply:
The thought is to not render anything for the WorkflowEditor when on the JobEditor view. 
Hence, if you're in the JobEditor and then decide to refresh your page. nothing from WorkflowEditor gets rendered.

Reply:
Right so it's an optimisation, nothing more? The fact that it was rendering wasn't getting in your way right?

Reply:
I'm a big fan of this change! And it's only possible because the store has been properly decoupled. I've not tested yet and I can't speak to the condition itself, but I hope we can keep it!

Reply:
> Right so it's an optimisation, nothing more? The fact that it was rendering wasn't getting in your way right?

Yeah. It's just optimization.